class/deeplink-schema-match.ts

```
@@ -380,10 +391,12 @@ class DeeplinkSchemaMatch {
             }
           } else if (value.startsWith('lightning')) {
             const lnpart = txInfo[index + 1].split('&').find(el => el.toLowerCase().startsWith('ln'));
-            lndInvoice = `lightning:${lnpart}`;
-            if (!this.isLightningInvoice(lndInvoice)) {
-              lndInvoice = false;
-              break;
+            if (lnpart) {
+              lndInvoice = `lightning:${lnpart}`;
+              if (!this.isLightningInvoice(lndInvoice)) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)